.gitignore

```diff
@@ -1,11 +1,14 @@
+.idea
```

Comment:
this does not belong in the .gitignore file, but in your global gitignore configuration.
http://devoh.com/blog/2013/01/global-gitignore